app/main.py

```diff
@@ -2,5 +2,18 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrapper(*args, **kwargs) -> Callable:
+
+        cache_key = args + tuple(sorted(kwargs.items()))
+        if cache_key in cache_dict:
+            print("Getting from cache")
+            return cache_dict[cache_key]
+        else:
```

Comment:
There is no need in the `else`, statement, code will not execute after `return` anyway. Remove `else`.